src/catalog/src/durable.rs

```diff
@@ -284,7 +286,10 @@ pub async fn shadow_catalog_state(
     organization_id: Uuid,
 ) -> impl OpenableDurableCatalogState {
     let stash = Box::new(stash_backed_catalog_state(stash_config));
-    let persist = Box::new(persist_backed_catalog_state(persist_client, organization_id).await);
+    let persist = Box::new(
+        persist_backed_catalog_state(persist_client, organization_id, &MetricsRegistry::new())
```

Comment:
ditto here: I'd pass down the MetricsRegistry. in general, I try to only construct them in main files and things that are themselves obviously test code

Reply:
Done